metaflow/metaflow_config.py

```diff
@@ -109,6 +109,12 @@
 # top-level retries)
 S3_TRANSIENT_RETRY_COUNT = from_conf("S3_TRANSIENT_RETRY_COUNT", 20)
 
+# S3 retry configuration used in the aws client
+# Use the adaptive retry strategy by default
+S3_CLIENT_RETRY_CONFIG = from_conf(
```

Comment:
what's the rationale for pulling this out?

Reply:
so it's shared between aws_client.py and s3op.py